docker/localnet/import-keys.sh

```diff
@@ -28,3 +28,9 @@ import_key "base_vesting_account" "coach index fence broken very cricket someone
 import_key "continuous_vesting_account" "phone worry flame safe panther dirt picture pepper purchase tiny search theme issue genre orange merit stove spoil surface color garment mind chuckle image"
 import_key "delayed_vesting_account" "pilot text keen deal economy donkey use artist divide foster walk pink breeze proud dish brown icon shaft infant level labor lift will tomorrow"
 import_key "periodic_vesting_account" "want merge flame plate trouble moral submit wing whale sick meat lonely yellow lens enable oyster slight health vast weird radar mesh grab olive"
+
+# import validator keys
+import_key "operator-0" "mix around destroy web fever address comfort vendor tank sudden abstract cabin acoustic attitude peasant hospital vendor harsh void current shield couple barrel suspect"
+import_key "operator-1" "useful case library girl narrow plate knee side supreme base horror fence tent glass leaf okay budget chalk patch forum coil crunch employ need"
+import_key "operator-2" "slight oblige answer vault project symbol dismiss match match honey forum wood resist exotic inner close foil notice onion acquire sausage boost acquire produce"
+import_key "operator-3" "prefer spring subject mimic shadow biology connect option east dirt security surge thrive kiwi nothing pulse holiday license hub pitch motion sunny pelican birth"
```

Comment:
make sure this mnemonic matching here
https://github.com/cheqd/cheqd-node/pull/792/files#diff-831d3f4a4deca212d21747e4207cd18a7f5db687c4e6200933095417fb02291aR192